src/openforms/authentication/contrib/demo/plugin.py

```diff
@@ -14,13 +14,21 @@
 from openforms.forms.models import Form
 from openforms.utils.validators import BSNValidator
 
-from ...constants import FORM_AUTH_SESSION_KEY, AuthAttribute
+from ...base import BasePlugin
```

Comment:
On line 13 there is still `from openforms.authentication.base import BasePlugin` which should be removed

Reply:
yeah, I spotted it in `feature/296-co-sign-formio-component` and fixed it there! Thanks for pointing it out though.